sheet.go

```diff
@@ -707,6 +708,59 @@ func (f *File) SearchSheet(sheet, value string) []string {
 	return result
 }
 
+// RegSearchSheet provides the ability to retrieve coordinates
+// with the given worksheet name and regular expression
+// For a merged cell, get the coordinates
+// of the upper left corner of the merge area.
+// :example)
+// Search the coordinates where the numerical value in the range of "0 - 9" of Sheet 1 is described:
+//
+//    xlsx.RegSearchSheet("Sheet1", "[0-9]")
+//
+// return map[CELL:result CELL:result ...]
+func (f *File) RegSearchSheet(sheet, value string) map[string]string {
```

Comment:
I suggest that the return value of the function `RegSearchSheet` use type `[][]string` instead of `map[string]string` will be better, for example: `[][]string{"B3", "cell value"}`.